packages/client/src/views/GrantDetailsView.vue

```diff
@@ -219,7 +219,7 @@ import GrantActivity from '@/components/GrantActivity.vue';
 
 const HEADER = '__HEADER__';
 const FAR_FUTURE_CLOSE_DATE = '2100-01-01';
-const NOT_AVAILABLE_TEXT = 'Not available';
+const NOT_AVAILABLE_TEXT = 'Not Available';
```

Comment:
Should this be "Not yet issued"?

[Figma link](https://www.figma.com/design/N1wuvwvciNhdKZNSdomTqn/Forecasted?node-id=123-1170&node-type=frame&t=jxt5pbcKPWp53xZ2-0)

![image](https://github.com/user-attachments/assets/91661d4f-e24b-45f4-90cd-30e32d6cb4ed)